tests/basic/gfapi/gfapi-copy-file-range.t

```diff
@@ -65,12 +75,20 @@ TEST stat $M0/new
 
 # check the size of the destination file
 DST_SIZE=$(stat -c %s $M0/new);
+DISC_USED_AFTER=`df --output=used $M0/file | tail -n1`
 
 # The sizes of the source and destination should be same.
 # Atleast it ensures that, copy_file_range API is working
 # as expected. Whether the actual cloning happened via reflink
 # or a read/write happened is different matter.
 TEST [ $SRC_SIZE == $DST_SIZE ];
+# Check if reflink feature is functional, indicated by disk usage remaining the
+# same after duplicating the file.
+TEST [ $DISC_USED_BEFORE == $DISC_USED_AFTER ];
```

Comment:
This depends on XFS. I'm not sure if we really need to test it. Maybe the kernel version doesn't support it, but we just need to be sure that functionally Gluster does what's expected, even if XFS implements it as a copy.

Reply:
Yep, I remove tihis check.